chia/data_layer/data_store.py

```diff
@@ -1206,11 +1250,11 @@ async def get_keys(
                 return []
 
             kv_ids = merkle_blob.get_keys_values()
+            raw_key_ids = [KeyOrValueId(id.raw) for id in kv_ids.keys()]
+            table_blobs = await self.get_table_blobs(raw_key_ids, store_id)
```

Comment:
maybe worth changing to `async def get_table_blobs(self, kv_ids: list[Union[KeyOrValueId, KeyId, ValueId]]` to avoid the conversions?  though i can't say i know offhand what sqlite does with a `KeyId` directly...  anyways, maybe close this after reading and we'll consider this separately.  just seems like it would be nice to be able to just pass `kv_ids.keys()` and be done.

Reply:
I went for this because sqlite doesn't like `KeyId` or `ValueId` - it throws while being exposed to them